### PR TITLE
Update task list with finalized FR and EN contents

### PR DESCRIPTION
**Pull request by ZhuZhuThanou**
### Description
Update finalized contents for tasks in FR and EN

List of proposed changes:
- Changes in text and additional tasks provided by the designer team

### What to test for/How to test
- npm i
- npm run dev
- answer questions, check console, there shouldn't be any warning when constructing the task list

### Additional Notes

### CODE REVIEW

File: frontend/src/data/tasks.json [hunk 1]
```diff
@@ -644,7 +644,7 @@
...
+        "descriptionFr": "Si vous souhaitez fractionner votre revenu de pension admissible avec votre conjoint ou partenaire, vous devrez tous deux remplir et soumettre un choix conjoint de fractionnement du revenu de pension privée au moment de remplir votre déclaration de revenus. **Cela ne s'applique pas à vos pensions de la SV et du RPC**.\n\nVous devrez réunir les renseignements suivants pour demander le fractionnement du revenu de pension\u00a0:\n- les feuillets\u00a0T4A du conjoint ou partenaire qui effectue le transfert\n- Les feuillets\u00a0T4A-RCA du conjoint ou partenaire qui effectue le transfert (le cas échéant)\n- Montant des prestations d'ancien combattant du conjoint ou partenaire qui effectue le transfert (le cas échéant)\n\nVous pouvez discuter avec un expert financier des conséquences du fractionnement du revenu de pension avant de déposer votre demande.",
```
Comment (sebastien-comeau) [inline frontend/src/data/tasks.json:647]: Non-breaking space `\u00a0` is not needed with `feuillets T4A` but change `-` for a non-breaking hypen `\u2011` with `T4A-RCA`.

File: frontend/src/data/tasks.json [hunk 2]
```diff
@@ -1142,8 +1124,8 @@
...
+        "descriptionEn": "If you continue working, this may affect the amount of tax you need to pay on your income.\n\nIf you can afford to delay collecting your public pensions, the longer you wait, the higher your monthly amounts will be for life.\n\nIf you continue to work while collecting your CPP retirement pension, and are under age 70, you can continue to take part in the CPP. Your CPP contributions will go toward post-retirement benefits (PRB), which will increase your monthly pension amount.",
+        "descriptionFr": "Si vous continuez à travailler, cela peut avoir une incidence sur le montant de l'impôt que vous devez payer sur vos revenus.\n\nSi vous pouvez vous le permettre, il est préférable de retarder le paiement de votre pension; plus vous attendez, plus vos montants mensuels seront élevés jusqu'à la fin de votre vie.\n\nSi vous continuez à travailler tout en recevant votre pension de retraite du RPC et que vous avez moins de 70\u00a0ans, vous pouvez continuer à cotiser au RPC. Vos cotisations seront applicables aux prestations après-retraite (PAR), ce qui augmentera votre revenu de retraite.",
```
Comment (sebastien-comeau) [inline frontend/src/data/tasks.json:1128]: Non-breaking space `\u00a0` is not needed with `70 ans`.

File: frontend/src/data/tasks.json [hunk 3]
```diff
@@ -1167,13 +1149,13 @@
...
+        "descriptionFr": "De plus en plus de Canadiens décident de travailler ou de gagner de l'argent pendant leur retraite. Si vous gagnez de l'argent pendant que vous touchez votre pension de la Sécurité de la vieillesse (SV), vous devez tenir compte de certains éléments importants\u00a0:\n\n- \u00a0Votre admissibilité au Supplément de revenu garanti dépend de votre revenu. Si vos revenus dépassent un certain montant, vous n'aurez pas droit à cette prestation.\n- Si votre revenu net de toutes provenances est supérieur à un montant seuil (81\u00a0761\u00a0$ pour 2022), vous devez rembourser une partie ou la totalité de votre pension de la SV. Votre revenu net de toutes provenances comprend tous les revenus que vous recevez dans n'importe quel pays, y compris les revenus de retraite provenant de vos pensions et de vos REER.",
```
Comment (sebastien-comeau) [inline frontend/src/data/tasks.json:1158]: Non-breaking space `\u00a0` is not needed before `Votre admissibilité`.

File: frontend/src/data/tasks.json [hunk 4]
```diff
@@ -1209,7 +1191,7 @@
...
```
Comment (sebastien-comeau) [inline frontend/src/data/tasks.json:1194]: Non-breaking space `\u00a0` is not needed before `Votre admissibilité`.

File: frontend/src/data/tasks.json [hunk 5]
```diff
@@ -1245,7 +1227,7 @@
...
+        "descriptionFr": "Le fait de passer du temps à l'étranger peut avoir des conséquences sur les prestations de la Sécurité de la vieillesse et du Régime de pensions du Canada (RPC) que vous recevez.\n\nVous pouvez continuer à recevoir vos pensions de l'État à l'étranger, mais le gouvernement du Canada peut retenir des impôts pouvant aller jusqu'à 25\u00a0%.\n\nSi vous vivez à l'extérieur du Canada pendant plus de six mois et que vous recevez le Supplément de revenu garanti (SRG), vous cesserez de recevoir cette prestation jusqu'à votre retour.\n\nVous ne pouvez pas percevoir l'allocation au survivant si vous êtes à l'étranger pendant plus de six mois.\n\nSi vous prévoyez de quitter le Canada pour une période de plus de six\u00a0mois, vous devez [communiquer avec Service Canada] (https://www.canada.ca/fr/contact/communiquez-avec-1-800-o-canada.html) pour éviter un plus-payé.",
```
Comment (sebastien-comeau) [inline frontend/src/data/tasks.json:1230]: Non-breaking space `\u00a0` is not needed with `six mois`.

File: frontend/src/data/tasks.json [hunk 6]
```diff
@@ -1259,7 +1241,7 @@
             "linkEn": "https://www.canada.ca/en/services/benefits/publicpensions/cpp/cpp-international/while-receiving.html",
             "linkFr": "https://www.canada.ca/fr/services/prestations/pensionspubliques/rpc/rpc-internationales/pendant-que-vous-recevez.html",
             "textEn": "Lived or living outside Canada - Pensions and benefits: While on pension and benefits",
-            "textFr": "Personnes ayant vécu ou vivant à l'étranger - Pensions et prestations Pendant que vous recevez pensions et prestations"
+            "textFr": "Personnes ayant vécu ou vivant à l'étranger – Pensions et prestations Pendant que vous recevez pensions et prestations"
```
Comment (sebastien-comeau) [inline frontend/src/data/tasks.json:1244]: Change this hypen type `–` with a regular hypen `-`.

File: frontend/src/data/tasks.json [hunk 7]
```diff
@@ -1287,7 +1269,7 @@
...
+        "descriptionFr": "Le fait de passer du temps à l'étranger peut avoir des conséquences sur les prestations de la Sécurité de la vieillesse et du Régime de pensions du Canada (RPC) que vous recevez.\n\nVous pouvez continuer à recevoir vos pensions de l'État à l'étranger, mais le gouvernement du Canada peut retenir des impôts pouvant aller jusqu'à 25\u00a0%.\n\nSi vous vivez à l'extérieur du Canada pendant plus de six mois et que vous recevez le Supplément de revenu garanti (SRG), vous cesserez de recevoir cette prestation jusqu'à votre retour.\n\nVous ne pouvez pas percevoir l'allocation au survivant si vous êtes à l'étranger pendant plus de six mois.\n\nSi vous prévoyez de quitter le Canada pour une période de plus de six\u00a0mois, vous devez [communiquer avec Service Canada] (https://www.canada.ca/fr/contact/communiquez-avec-1-800-o-canada.html) pour éviter un plus-payé.",
```
Comment (sebastien-comeau) [inline frontend/src/data/tasks.json:1272]: Non-breaking space `\u00a0` is not needed with `six mois`.

File: frontend/src/data/tasks.json (hunk 1 above)
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:647]: removed `\u00a0` before feuillets T4A and add T4A`\u2011`RCA

File: frontend/src/data/tasks.json [hunk 8]
```diff
@@ -740,7 +740,7 @@
             "linkEn": "https://www.canada.ca/en/public-health/services/publications/healthy-living/physical-activity-tips-older-adults-65-years-older.html",
             "linkFr": "https://www.canada.ca/fr/sante-publique/services/publications/vie-saine/conseils-activite-physique-aines-65-ans.html",
             "textEn": "Physical activity tips for older adults (65 years and older)",
-            "textFr": "Conseils pratiques pour les aînés (65 ans et plus)"
+            "textFr": "Conseils pratiques pour les aînés (65\u00a0ans et plus)"
```
[the remark this answers is not in the document]
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:743]: removed `\u00a0`

File: frontend/src/data/tasks.json (hunk 2 above)
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1128]: Removed `\u00a0`

File: frontend/src/data/tasks.json (hunk 3 above)
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1158]: removed `\u00a0 ` in front of `Votre admissibilité`

File: frontend/src/data/tasks.json (hunk 4 above)
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1194]: removed `\u00a0` in front of `Votre admissibilité`

File: frontend/src/data/tasks.json (hunk 6 above)
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1244]: Changed `–` to `-`

File: frontend/src/data/tasks.json (hunk 7 above)
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1272]: remove `\u00a0` in front of `six mois.`

File: frontend/src/data/tasks.json [hunk 9]
```diff
@@ -1329,7 +1311,7 @@
...
+        "descriptionFr": "Le fait de passer du temps à l'étranger peut avoir des conséquences sur les prestations de la Sécurité de la vieillesse et du Régime de pensions du Canada (RPC) que vous recevez.\n\nVous pouvez continuer à recevoir vos pensions de l'État à l'étranger, mais le gouvernement du Canada peut retenir des impôts pouvant aller jusqu'à 25\u00a0%.\n\nSi vous vivez à l'extérieur du Canada pendant plus de six mois et que vous recevez le Supplément de revenu garanti (SRG), vous cesserez de recevoir cette prestation jusqu'à votre retour.\n\nVous ne pouvez pas percevoir l'allocation au survivant si vous êtes à l'étranger pendant plus de six mois.\n\nSi vous prévoyez de quitter le Canada pour une période de plus de six\u00a0mois, vous devez [communiquer avec Service Canada] (https://www.canada.ca/fr/contact/communiquez-avec-1-800-o-canada.html) pour éviter un plus-payé.",
```
[the remark this answers is not in the document]
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1314]: remove `\u00a0` between `six` and  `mois.`

File: frontend/src/data/tasks.json [hunk 10]
```diff
@@ -1371,7 +1353,7 @@
...
```
[the remark this answers is not in the document]
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1356]: removed `\u00a0` between `six` and `mois`

File: frontend/src/data/tasks.json [hunk 11]
```diff
@@ -1694,7 +1676,7 @@
...
-        "descriptionFr": "Si vous êtes membre d'une Première Nation, Métis ou Inuit, et que vous avez travaillé à l'extérieur de la réserve, vous avez le droit de recevoir une pension de retraite du Régime de pensions du Canada (RPC).\n\nSi vous n'avez travaillé que dans une réserve, il est possible que vous n'ayez pas cotisé au RPC. Si vous ne le savez pas, vous pouvez demander une copie de votre état de compte du cotisant au RPC à Service Canada (MDSC).",
+        "descriptionFr": "Si vous êtes membre d'une Première Nation, Métis ou Inuit, et que vous avez travaillé à l'extérieur de la réserve, vous avez le droit de recevoir une pension de retraite du Régime de pensions du Canada (RPC).\n\nSi vous n'avez travaillé que dans une réserve, il est possible que vous n'ayez pas cotisé au RPC. Si vous ne le savez pas, vous pouvez demander une copie de votre état de compte du cotisant au RPC à Service\u00a0Canada (MDSC).",
```
[the remark this answers is not in the document]
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1679]: removed `\u00a0` between `Service ` and `Canada`

File: frontend/src/data/tasks.json [hunk 12]
```diff
@@ -1526,7 +1508,7 @@
...
-        "descriptionFr": "Vous n'avez pas droit à la SV si vous n'avez pas vécu au Canada pendant au moins 10 ans après l'âge de 18 ans. Vous y aurez droit dès que vous aurez vécu suffisamment longtemps au Canada. Calculez le temps que vous avez passé au Canada pour savoir quand présenter votre demande. Si vous avez vécu dans certains pays ayant conclu des accords de sécurité sociale avec le Canada, vous pouvez y avoir droit plus tôt. Suivez le lien ci-dessous pour savoir si votre pays a conclu un accord avec le Canada.",
+        "descriptionFr": "Vous n'avez pas droit à la SV si vous n'avez pas vécu au Canada pendant au moins 10\u00a0ans après l'âge de 18\u00a0ans. Vous y aurez droit dès que vous aurez vécu suffisamment longtemps au Canada. Calculez le temps que vous avez passé au Canada pour savoir quand présenter votre demande. Si vous avez vécu dans certains pays ayant conclu des accords de sécurité sociale avec le Canada, vous pouvez y avoir droit plus tôt. Suivez le lien ci-dessous pour savoir si votre pays a conclu un accord avec le Canada.",
```
[the remark this answers is not in the document]
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1511]: remove `\u00a0` between `10` and  `ans`,  `18` and `ans`

File: frontend/src/data/tasks.json (hunk 5 above)
Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1230]: removed `\u00a0` between `six` and `mois`

File: frontend/src/data/tasks.json
Comment (sebastien-comeau) [inline frontend/src/data/tasks.json:1]: - Remove non-breaking spaces with `{NUMBER}e anniversaire` and `{NUMBER} ans`.
- Fix invalid [markdown links](https://www.markdownguide.org/basic-syntax/#links). No space allowed between `]` and `(`.

Comment (ZhuZhuThanou) [reply frontend/src/data/tasks.json:1]: Search and replace pattern `{NUMBER}e\u00a0anniversaire` with `{NUMBER}e anniversaire`
Search are replace pattern `{NUMBER}\u00a0ans` with `{NUMBER} ans`

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**